### ISSUE TITLE
Is there any simple way to annotate plots in Forge?

### ISSUE THREAD

**Issue by 3togo** [arrayfire/forge#237]
Is there any simple way to annotate plots in Forge?  something like add a marker with a text

**Comment by 9prady9** [comment]
Yes, please check the examples https://github.com/arrayfire/forge/tree/master/examples

**Comment by 9prady9** [comment]
If you are referring to annotations like these https://github.com/arrayfire/forge/pull/238 then those aren't there yet. I will take a look at your PR and get back to you by EOW, thank you for submitting it.
